spec/rubocop/cop/dependency/over_boundary_spec.rb

```diff
@@ -158,10 +158,10 @@ class A
 
   context 'when complex Rules' do
     let(:cop_config) do
-      RuboCop::Config.new('Rules' => [{ 'BannedConsts' => 'Bar', 'FromNamespacePatterns' => '\AFoo\z' }])
+      RuboCop::Config.new('Rules' => [{ 'BannedConstPatterns' => '\ABar', 'FromNamespacePatterns' => '\AFoo\z' }])
```

Comment:
if there is no specific reason, it might be better to add an end-of-line pattern `\z`, similar to what's specified in `FromNamespacePatterns`.

```suggestion
      RuboCop::Config.new('Rules' => [{ 'BannedConstPatterns' => '\ABar\z', 'FromNamespacePatterns' => '\AFoo\z' }])
```

Reply:
Sorry, I reveted the commit 7713a4f17a9c130930d144fa12bc1bc88eb6bc0a because this commit make the test of `'registers an offense when referring const `Bar::X` from class `Foo`'` failed.

Reply:
Should I make it '\ABar\z' and delete this failed test?

I'm starting to feel that the following two test cases are same...

I feel like I'm testing Regexp rather than testing this Gem.

```rb
it 'registers an offense when referring const `Bar` from class `Foo`' do
  expect_offense(<<~RUBY)
    class Foo
      Bar.new
      ^^^ Const `Bar` cannot use from namespace `Foo`.
    end
  RUBY
end

it 'registers an offense when referring const `Bar::X` from class `Foo`' do
  expect_offense(<<~RUBY)
    class Foo
      Bar::X.new
      ^^^^^^ Const `Bar::X` cannot use from namespace `Foo`.
    end
  RUBY
end
```

Reply:
You make sense. It is OK to add / delete tests anymore!